api/main_endpoints/routes/OfficeAccessCard.js

```diff
@@ -55,8 +80,15 @@ router.get('/verify', async (req, res) => {
 
   const cardExists = await checkIfCardExists(cardBytes);
 
+  const cardVerify = await verifiedCountCard(cardBytes); // cardExists before the changes
+
   if (cardExists) {
-    return res.sendStatus(OK);
+    // return res.sendStatus(OK); Commenting this out for now
+    // Going to add it
+    return {
+      'Status':res.sendStatus(OK),
+      'CardVerify':cardVerify
+    };
```

Comment:
lets. keep the original empty 200 response, the esp32 doesnt care about the card data we stored